extensions/eclipse-che-theia-file-sync-tracker/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@eclipse-che/theia-file-sync-tracker",
+  "keywords": [
+    "theia-extension"
+  ],
+  "version": "1.0.0",
```

I think version should be 
 "version": "0.0.1" as other extensions

```suggestion
  "version": "0.0.1",
```